youtube_dl/extractor/soundcloud.py

```diff
@@ -217,8 +217,15 @@ def _extract_info_dict(self, info, full_title=None, quiet=False, secret_token=No
         if quiet:
             self.report_extraction(name)
         thumbnail = info.get('artwork_url') or info.get('user', {}).get('avatar_url')
+        thumbnails = []
         if isinstance(thumbnail, compat_str):
-            thumbnail = thumbnail.replace('-large', '-t500x500')
+            thumbnail = thumbnail.replace('-large', '-original')
```

Comment:
All three thumbnails should be extracted.

Reply:
Fixed to extract the 3 known formats used on Soundcloud’s page, but there may be other unlisted formats available